test/int/federated_auth_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/test/helper/resources"
 )
 
-var _ = Describe("AtlasFederatedAuth test", Label("AtlasFederatedAuth", "federated-auth"), func() {
+var _ = FDescribe("AtlasFederatedAuth test", Label("AtlasFederatedAuth", "federated-auth"), func() {
```

Comment:
Looks like a left-over from the local testing :)

Reply:
We need a rule to catch this in the linter :grimacing: